api/tests/unit/domain/validators/campaign-validator_test.js

```diff
@@ -288,6 +290,47 @@ describe('Unit | Domain | Validators | campaign-validator', function () {
             });
           });
 
+          context('on multipleSendigs attribute', function () {
+            // given
+            const expectedBooleanError = {
+              attribute: 'multipleSendings',
+              message: '"multipleSendings" must be a boolean',
```

Comment:
Merger les 2 erreurs, cela n'apporte aucune valeur de différencier les 2
```suggestion
              message: 'MULTIPLE_SENDINGS_CHOICE_IS_REQUIRED',
```